bookwyrm/urls.py

```diff
@@ -33,6 +33,10 @@
         "robots.txt",
         TemplateView.as_view(template_name="robots.txt", content_type="text/plain"),
     ),
+    path(
+        "manifest.json",
+        TemplateView.as_view(template_name="manifest.json", content_type="application/json"),
```

Comment:
running `./bw-dev black` will fix the pylint error by slightly adjusting the layout of this path.

Reply:
Fixed.